bedrock/settings/base.py

```diff
@@ -1466,6 +1493,12 @@ def facebook_tab_url_lazy():
 # Optimize.ly project code
 OPTIMIZELY_PROJECT_ID = None
 
+# Fx Accounts Relier Service Client ID
+# TODO: Update when available! Currently using creds for demo5 & URLs for FxA dev
```

Comment:
Yeah, there are two bugs - one to provide creds and one to put them on prod/stage/dev. They are both in progress and should (hopefully) be done today.

https://bugzilla.mozilla.org/show_bug.cgi?id=1131660
https://bugzilla.mozilla.org/show_bug.cgi?id=1131781